src/main/flight/pid.c

```diff
@@ -872,6 +872,10 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 
     rotateItermAndAxisError();
 
+#ifdef USE_RPM_FILTER
+￼    rpmFilterUpdate();
```

Comment:
special character typo

Reply:
Damn - GitHub copy / paste isn't working again. :-( Fixed, thanks for letting me know.